src/containers/Form.js

```diff
@@ -0,0 +1,173 @@
+				'20b': '',
+				'21a': null,
+				'21b': '',
+				'22a': [],
+				'22b': '',
+				'22c': [],
+				'22d': '',
+				23: [],
+				'24a': [],
+				'24b': '',
+				'25a': null,
+				'25b': [],
+				'26a': [],
+				'26b': ''
+			},
+			complainantInfo: {
+				'27a': '',
+				'27b': '',
+				'27c': '',
+				'27d': ''
+			},
+			dispositionInfo: {
+				'28a': [],
+				'28b': [],
+				'28c': '',
+				'29a': [],
+				'29b': '',
+				30: [],
+				31: ''
+			},
+			officerInfo: {
+				32: '',
+				34: '',
+				35: '',
+				36: [],
+				'37a': '',
+				'37b': ''
+			},
+			submitSuccess: null,
+			submitFailure: null
+		};
+
+		this.handleInput = this.handleInput.bind(this);
+		this.handleSingleSelection = this.handleSingleSelection.bind(this);
+		this.handleCheckboxChange = this.handleCheckboxChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	// For text input
+	handleInput(e) {
+		const sectionKey = e.target.dataset.section
+		const name = e.target.name;
+		const input = e.target.value;
+		const sectionState = this.state[sectionKey]; 
+		sectionState[name] = input;
+		this.setState({ [sectionKey]: sectionState }, () => {console.log('section state:', this.state[sectionKey])});
+	}
+
+	// For radio or select input
+	handleSingleSelection(e) {
+		const sectionKey = e.target.dataset.section;
+		const sectionState = this.state[sectionKey];
+		sectionState[e.target.name] = e.target.value;
+		this.setState({ [sectionKey]: sectionState }, () => {console.log('section state:', this.state[sectionKey])});
+	}
+
+	handleCheckboxChange(e) {
+		const sectionKey = e.target.dataset.section;
+		const sectionState = this.state[sectionKey];
+		const idx = sectionState[e.target.name].indexOf(e.target.value);
+		console.log('checkbox sectionState before addition', sectionState);
+		if (idx === -1) {
+			console.log('does not exist');
+			sectionState[e.target.name].push(e.target.value);
+		} else {
+			console.log('does exist, idx:', idx);
+			sectionState[e.target.name].splice(idx, 1);
+		}
+		console.log('checkbox sectionState before saving:', sectionState);
+		this.setState({ [sectionKey]: sectionState }, () => {console.log('section state:', this.state[sectionKey])});
+	}
+
+	handleSubmit(e) {
+		e.preventDefault();
+		console.log('SUBMIT:', this.state);
+		// const entities = Object.assign({}, this.state.reportInfo, this.state.consumerInfo, this.state.complainantInfo, this.state.dispositionInfo, this.state.officerInfo);
+
+		// 'Pillows' test entity set id
+		const entitySetId = '78075393-3449-434d-a41e-0e30d42ba96f';
+		const entities = {
+			// color property type id
+			'01703452-0cd6-4bb6-a8f4-ba2d8ef43c26': 'yellow',
+			// texture property type id
+			'135f73d5-4d91-4735-97bc-09fb1e72555b': 'sharp'
+		}
+		console.log('entities:', entities);
+
+		EntityDataModelApi.getEntitySetId('pillows')
+		.then((id) => {
+			return DataApi.createEntityData(id, '', entities);
+		})
+		.then((res) => {
+			console.log('success! res:', res);
```

Comment:
This is ok for now, but when this is ready to go to prod, you'll have to get rid of `console.log`. If you want, you can use the `Logger` utility from `lattice-js`, which can make this much easier. We can talk about this later.